checkerConstants.cpp

```diff
@@ -0,0 +1,61 @@
+// Constant declarations
+#include "checkerTypes.h"
+
+namespace BMSVariables{
+  // Float values
+  float minTemperature = 0.0;
+  float maxTemperature = 45.0;
+  float maxChargeRate  = 0.8;
+  float minSOC = 20.0;
+  float maxSOC = 80.0;
+  float tempTolerancePercent   = 5.0;
+  float socTolerancePercent    = 5.0;
+  float cRateTolerancePercent  = 5.0;
+
+  // Variable Strings
+  languageIndexer temperatureStrings{
```

Comment:
Good isolation of strings in a file.
Consider this: If we add Spanish, then we would add a third string to each of these maps. Suppose we make a mistake and leave one of the maps (like chargeRateStrings) without the third string.
The program would throw an exception when someone chooses Spanish.
Think of a test that would catch this error without having to run the whole program with each different language.